---
title: LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER |Microsoft ドキュメント
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.prod_service: database-engine
ms.service: ''
ms.component: localdb
ms.reviewer: ''
ms.suite: sql
ms.technology: ''
ms.tgt_pltfrm: ''
ms.topic: reference
ms.assetid: c4288265-43f7-44fd-acab-ba64d04264d9
caps.latest.revision: 10
author: stevestein
ms.author: sstein
manager: craigg
ms.workload: Inactive
ms.openlocfilehash: c6ef82c14f3ff941275e6e0f1e488f53e6300cde
ms.sourcegitcommit: 7a6df3fd5bea9282ecdeffa94d13ea1da6def80a
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 04/16/2018
---
# <a name="localdberrorcannotaccessinstancefolder"></a>LOCALDB_ERROR_CANNOT_ACCESS_INSTANCE_FOLDER
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|261|  
|イベント ソース|SQL Server Local Database Runtime 12.0|  
|コンポーネント|Local Database Runtime API|  
|メッセージ テキスト|ローカル データベース インスタンスのフォルダーにアクセスできません: %%LOCALAPPDATA%%\Microsoft\Microsoft SQL Server のローカル DB\Instances\\< インスタンス名\>です。|  
  
## <a name="explanation"></a>説明  
 インスタンス フォルダーにアクセスできません。  
  
## <a name="user-action"></a>ユーザーの操作  
  
